starfish/experiment/builder/__init__.py

```diff
@@ -35,17 +37,40 @@
 DEFAULT_DIMENSION_ORDER = (Axes.ZPLANE, Axes.ROUND, Axes.CH)
 
 
+class ReadOnlyTile(io.BytesIO):
```

Comment:
I wouldn't call this `ReadOnlyTile`.  It's more like a `/dev/null/`.

Furthermore, I would stub out the `read` and `seek` methods because both of them won't behave as one might expect.